Doc/reference/datamodel.rst

```diff
@@ -1006,7 +1006,7 @@ Special attributes:
       a :ref:`generic class <generic-classes>`.
 
    :attr:`~class.__static_attributes__`
-      A tuple containing names of attributes of this class which are accessed
+      A tuple containing names of attributes of this class which are modified
```

Comment:
Maybe "assigned to" rather than "modified through".

Reply:
It could be del'ed as well.

Reply:
Ah I just saw the other comment. Ok.